### PR TITLE
[EPMRPP-41738] Update data store for thumbnails

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/binary/impl/AttachmentDataStoreService.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+@Service("attachmentDataStoreService")
+public class AttachmentDataStoreService extends CommonDataStoreService {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/binary/impl/AttachmentDataStoreService.java]: tests

File: src/main/java/com/epam/ta/reportportal/binary/impl/UserDataStoreService.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+@Service("userDataStoreService")
+public class UserDataStoreService extends CommonDataStoreService {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/binary/impl/UserDataStoreService.java]: tests

File: src/main/java/com/epam/ta/reportportal/binary/impl/AttachmentDataStoreService.java (hunk 1 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/binary/impl/AttachmentDataStoreService.java]: done

File: src/main/java/com/epam/ta/reportportal/binary/impl/UserDataStoreService.java (hunk 2 above)
Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/binary/impl/UserDataStoreService.java]: done

**Pull request merged**